easybuild/easyconfigs/h/Horovod/Horovod-0.21.0-CrayGNU-21.09-tf-2.4.0.eb

```diff
@@ -0,0 +1,34 @@
+easyblock = 'PythonPackage'
+
+name = 'Horovod'
+version = '0.21.0'
+versionsuffix = '-tf-2.4.0-xxx'
+
+homepage = 'https://github.com/uber/horovod'
+description = "Horovod is a distributed training framework for TensorFlow."
+
+toolchain = {'name': 'CrayGNU', 'version': '21.09'}
+toolchainopts = {'verbose': False}
+
+source_urls = [PYPI_LOWER_SOURCE]
+sources = [SOURCELOWER_TAR_GZ]
+
+dependencies = [
+    ('cray-python', EXTERNAL_MODULE),
+    ('cudatoolkit', EXTERNAL_MODULE),
+    ('numpy', '1.17.2'),
```

Comment:
@rsarm The module `cray-python` already provides `numpy` 1.20.1; if you want to load a module that we build due to performance reasons, there is the more recent version 1.21.4 available with the corresponding easyconfig file